tests/test_enterprise/views/test_grant_data_sharing_permissions.py

```diff
@@ -401,29 +416,42 @@ def test_post_course_specific_consent(
             enable_data_sharing_consent=True,
             enforce_data_sharing_consent='at_enrollment',
         )
+        content_filter = {
+            'key': [
+                course_id,
+            ]
+        }
+        EnterpriseCustomerCatalogFactory(
+            enterprise_customer=enterprise_customer,
+            content_filter=content_filter
+        )
         ecu = EnterpriseCustomerUserFactory(
             user_id=self.user.id,
             enterprise_customer=enterprise_customer
         )
-        EnterpriseCourseEnrollment.objects.create(
+        enterprise_enrollment = EnterpriseCourseEnrollment.objects.create(
             enterprise_customer_user=ecu,
             course_id=course_id,
         )
         dsc = DataSharingConsentFactory(
             username=self.user.username,
             course_id=course_id,
             enterprise_customer=enterprise_customer,
-            granted=consent_provided
+            granted=False
```

Comment:
Was this change intentional?

Reply:
good catch, this was a leftover debug change